src/patternfly/components/AlertGroup/examples/AlertGroup.md

```diff
@@ -112,7 +112,13 @@ cssPrefix: pf-c-alert-group
 ### Overview
 An alert group that includes the `.pf-m-toast` modifier becomes a toast alert group with unique positioning in the top-right corner of the window. `.pf-c-alert-group` is required to create a toast alert group.
 
-Every toast alert must include a close button to dismiss the alert.
+Every toast alert must include a close button to dismiss the alert. <br><br>
```

Comment:
do you need these or could you just put a couple of new lines between this sentence and the next one?